src/app

```diff
@@ -52,43 +52,28 @@ if test -d "$PERSISTENT_STORE"; then
     fi
 fi
 
-CONFIG_HAS_XDELTA3_CMD="xdelta3"
-CONFIG_HAS_TAR_CMD="tar"
-delta_cmd="${CONFIG_HAS_XDELTA3_CMD} -d -s"
-tar_decompress_cmd="${CONFIG_HAS_TAR_CMD} -xzvf"
+delta_cmd="${XDELTA_COMMAND:-xdelta3}"
```

Comment:
I am sorry I missed that, but this is a breaking change: you removed `-d -s` flags, and thats why the tests fail. the problem with some of the testing is: with that sort of error in the code it is possible that the images were pulled during the test (I will try to find some time to address that)

Reply:
Right, I forgot to revert the delta flags. This was a misunderstanding on my end. Will revert.